docs/reference-1.0.0.html

```diff
@@ -1019,7 +1019,7 @@ <h4 id='map-methods-for-modifying-map-state'>Methods for modifying map state</h4
 </td>
 	</tr>
 	<tr id='map-setzoom'>
-		<td><code><b>setZoom</b>(<nobr>&lt;Number&gt;</nobr> <i>zoom</i>, <nobr>&lt;<a href='#zoom/pan-options'>Zoom/pan options</a>&gt;</nobr> <i>options</i>)</nobr></code></td>
+		<td><code><b>setZoom</b>(<nobr>&lt;Number&gt;</nobr> <i>zoom</i>, <nobr>&lt;<a href='#zoom/pan-options'>Zoom/pan options</a>&gt;</nobr> <i>options?</i>)</nobr></code></td>
```

Comment:
These are changes in the _compiled_ HTML docs - any changes here will be overwritten next time we compile the documentation.

The real documentation is inside the code, see [CONTRIBUTING.md](https://github.com/Leaflet/Leaflet/blob/master/CONTRIBUTING.md#improving-documentation) on improving docs.